test/integration/004_simple_snapshot_test/test-check-col-snapshots-bq/snapshot.sql

```diff
@@ -1,10 +1,10 @@
 {% snapshot snapshot_actual %}
     {# this used to be check_cols=('email',), which ought to be totally valid,
-    but isn't because type systems are hard. #}
+    but is not because type systems are hard. #}
     {{
         config(
             target_database=var('target_database', database),
-            target_schema=schema,
+            target_schema=var('target_schema', schema),
```

Comment:
To test that the aliasing works, could you provide the actual config names as `target_project` (instead of `target_database`) and `target_dataset` (instead of `target_schema`)?

Reply:
edited config to use alias as described